test/helpers/position_assertions.cc

```diff
@@ -30,6 +30,7 @@ const regex whitespaceRegex("^[ ]*$");
 const UnorderedMap<
     string, function<shared_ptr<RangeAssertion>(string_view, unique_ptr<Range> &, int, string_view, string_view)>>
     assertionConstructors = {
+        {"info", ErrorAssertion::make},
```

Comment:
This is all you've done to implement the `# info` assertion? Is there any other code in the test hardness for this?

If not, how are you guaranteeing that expected and actual severity matches?